nlp_controller.py

```diff
@@ -34,17 +35,18 @@
     "flair_sent_splitter": FlairSentSplitter,
     "s_typer": STyper,
     "dep_parser": DepParser,
+    "datetimerecognizer": DateTimeRecognizer,
     "xrenner": XrennerCoref,
     "flair_edu_splitter": FlairEDUSplitter,
-    "rst_parser": RSTParser,
+    "rst_parser": RSTParser
 }
 
 
 class NLPController:
     def __init__(self, opts):
         logging.info("Initializing NLP Controller...")
         opts.update(
-            {"SCRIPT_DIR": SCRIPT_DIR, "LIB_DIR": LIB_DIR, "BIN_DIR": BIN_DIR, "TT_PATH": TT_PATH,}
+            {"SCRIPT_DIR": SCRIPT_DIR, "LIB_DIR": LIB_DIR, "BIN_DIR": BIN_DIR, "TTG_PATH": TTG_PATH}
```

Comment:
could you update our two existing TT modules to handle this name change? There's a [tokenizer](https://github.com/gucorpling/amalgum/blob/master/nlp_modules/tt_tokenizer.py) and a [tagger](https://github.com/gucorpling/amalgum/blob/master/nlp_modules/tt_tagger.py)

Reply:
actually, looks like only the tagger needs to know where the binary is

Reply:
changed the paths in the tree tagger module so new binary is discoverable. Note that the binary is downloaded only during date/time processing, which is a later step in the pipeline.